### PR TITLE
clean up userimpact

### PR DESCRIPTION
**Pull request by amaliwanag2**

### CODE REVIEW

File: src/components/UserImpact.js [hunk 1]
```diff
@@ -213,7 +215,8 @@ const UserImpact = ({ user, disabled }) => {
         </MuiDialogTitle>
         <MuiDialogContent className={classes.themedLink}>
           <div className={classes.centerImage}>
-            <DolphinGif />
+            {/* TODO: @workaround/tab-generalization */}
```
Comment (amaliwanag2) [inline src/components/UserImpact.js:218]: will be resolved when I do S3 image refactor

File: src/components/UserImpactContainer.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { createFragmentContainer, graphql } from 'react-relay'
-import UserImpactSwitchComponent from 'src/components/UserImpactSwitchComponent'
+import UserImpactSeas from 'src/components/UserImpact'
```
Comment (kmjennison) [inline src/components/UserImpactContainer.js:2]: Rename UserImpactSeas to UserImpact

**Review by kmjennison — APPROVED** [review]
LGTM! 🦊

**Comment by kmjennison** [comment]
@amaliwanag2 I did not manually test, so relying on your confidence/testing here.

**Comment by amaliwanag2** [comment]
> @amaliwanag2 I did not manually test, so relying on your confidence/testing here.

sounds good.  I did heavy manual testing and found a couple small existing typos / issues.  So hopefully this is even better than parity

**Pull request merged**